cleanlab_studio/studio/trustworthy_language_model.py

```diff
@@ -61,8 +62,11 @@ def __init__(
                 f"Invalid quality preset {quality_preset} -- must be one of {_VALID_TLM_QUALITY_PRESETS}"
             )
 
+        self._return_log = False
         if options is not None:
             validate_tlm_options(options)
+            if "log" in options.keys():  # TODO: figure out check
```

Comment:
What is this TODO?

Reply:
It was a TODO to validate the input log options, `log` options are already validated [here](https://github.com/cleanlab/cleanlab-studio/blob/94bcf094a12db7897eb87477f80e0cf03b875a4a/cleanlab_studio/internal/tlm/validation.py#L184-L193), so I'll delete this